content/tutorial-svd.md

```diff
@@ -35,12 +35,12 @@ After this tutorial, you should be able to:
 
 ## Content
 
-In this tutorial, we will use a [matrix decomposition](https://en.wikipedia.org/wiki/Matrix_decomposition) from linear algebra, the Singular Value Decomposition, to generate a compressed approximation of an image. We'll use the `face` image from the [scipy.misc](https://docs.scipy.org/doc/scipy/reference/misc.html#module-scipy.misc) module:
+In this tutorial, we will use a [matrix decomposition](https://en.wikipedia.org/wiki/Matrix_decomposition) from linear algebra, the Singular Value Decomposition, to generate a compressed approximation of an image. We'll use the `face` image from the [scipy.datasets](https://docs.scipy.org/doc/scipy/reference/datasets.html) module:
 
 ```{code-cell}
-from scipy import misc
+from scipy import datasets
 
-img = misc.face()
+img = datasets.face()
```

Comment:
In additon to the previous change, GH didn't let me do only one suggestion due to the deleted line in the diff
```suggestion
img =face()
```